File: codetree/trail2/두_정수에_대한_연산값/operational-values-for-two-integers.py
import os
import sys

file_path = os.path.join(os.path.dirname(__file__), "input_test.txt")

if os.path.exists(file_path):
    sys.stdin = open(file_path, "r", encoding="utf-8")


# 📥 Input
def get_input_data() -> tuple[int, int]:
    a, b = map(int, input().split())

    return a, b


# ⚙️ Logic
def solution(a: int, b: int) -> tuple[int, int]:
    if a > b:
        a += 25
        b *= 2
    else:
        a *=2
        b += 25
    return a, b


# 🚀 Run Program
if __name__ == "__main__":
    print(*solution(*get_input_data()))

File: codetree/trail2/두 정수에 대한 연산값/operational-values-for-two-integers.ts
declare var require: any;
const fs: any = require("fs");

const filePath: string = fs.existsSync("./input_test.txt")
    ? "./input_test.txt"
    : "/dev/stdin";

const input: string[] = fs.readFileSync(filePath, "utf-8").trim().split(/\n+/);

/* 📥 Input */
const getInputData = (): [number, number] => {
    let idx: number = 0;
    const [a, b] = input[idx].split(' ').map(Number);
  
    return [a, b]
};

/* ⚙️ Logic */
const solution = (a: number, b: number): [number, number] => {
    if (a > b) {
        a += 25;
        b *= 2;
    }
    else {
        b += 25;
        a *= 2;
    }

    return [a, b];
};

/* 🚀 Run Program */
(() => {
    console.log(...solution(...getInputData())); 
})();